src/Libraries/Microsoft.Extensions.AI.Abstractions/Contents/ErrorContent.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Text.Json.Serialization;
+using Microsoft.Shared.Diagnostics;
+
+namespace Microsoft.Extensions.AI;
+
+/// <summary>
+/// Represents an error content.
+/// </summary>
+[DebuggerDisplay("{DebuggerDisplay,nq}")]
+public class ErrorContent : AIContent
```

Comment:
Is there anywhere in chat we should be using this as well?

Should we be modeling refusals using ErrorContent?

Reply:
I see `ErrorContent` being used as a non blocking answer. i.e:

Result was OK but during the process it stumbled into some errors. (Temporary data loss, device unplugged, unable to access internet in the backend, etc).

For the Refusal, since the model refused and turned down, an exception might be ideal.